ezyrb/mapper.py

```diff
@@ -0,0 +1,243 @@
+"""
+Utilities for mapping solutions on a different grid
```

Comment:
Please check the english in the documentation because there are a lot of mistakes and nonsense.